### PR TITLE
Bump ech_hrr_key length to 32 bytes to align with latest HPKE changes.

### PR DESCRIPTION
**Pull request by chris-wood**
See https://github.com/cfrg/draft-irtf-cfrg-hpke/pull/159, and the related [CFRG thread](https://mailarchive.ietf.org/arch/msg/cfrg/mxBzZq85ijSMNMUwyI3gA3B0U4U/).

cc @cjpatton

### CODE REVIEW

**Comment by cjpatton** [comment]
Looks good to me.

File: draft-ietf-tls-esni.md [hunk 1]
```diff
@@ -748,7 +748,7 @@ ECHConfig, as follows:
 ~~~
     context = SetupBaseR(ClientECH.enc, skR, "tls13 ech")
     ClientHelloInner = context.Open("", ClientECH.payload)
-    ech_hrr_key = context.Export("tls13 ech hrr key", 16)
+    ech_hrr_key = context.Export("tls13 ech hrr key", 32)
```
Comment (cjpatton) [inline draft-ietf-tls-esni.md:751]: You also need to change this above.

s/context.Export("tls13 ech hrr key", 16)/context.Export("tls13 ech hrr key", 32)

**Review by cjpatton — CHANGES_REQUESTED** [review]

File: draft-ietf-tls-esni.md (hunk 1 above)
Comment (chris-wood) [reply draft-ietf-tls-esni.md:751]: Grr, I thought caught them all. Thanks!

**Comment by cjpatton** [comment]
Looks good!

**Pull request merged**